src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -571,4 +571,13 @@
   <data name="DoNotRegisterCompilerTypesWithBadAssemblyReferenceRuleTitle" xml:space="preserve">
     <value>Compiler extensions should be implemented in assemblies with compiler-provided references</value>
   </data>
+  <data name="SemanticModelGetDeclaredSymbolAlwaysReturnsNullDescription" xml:space="preserve">
+    <value>Calling 'SemanticModel.GetDeclaredSymbol' with an argument of type 'GlobalStatementSyntax', 'IncompleteMemberSyntax' or a type inheriting from 'BaseFieldDeclarationSyntax' will always return 'null'.</value>
```

Comment:
It feels like fields should get a specific error message that instructs the user to extract the variable declarators.

Reply:
Seems like this description needs adjustment as fields now have a separate descriptor/description. Additionally, not sure if we should limit the description to just 'GlobalStatementSyntax' and 'IncompleteMemberSyntax' as the code handles all sub-types of `SyntaxNode` that do not have an overload. It's fine to mentioned these two as examples, but not as the only node types being handled.